docs/user/reference/metrics/custom_distribution.md

```diff
@@ -187,7 +258,11 @@ use glean_metrics::graphics;
 // Were any of the values negative and thus caused an error to be recorded?
 assert_eq!(
     0,
-    graphics::checkerboard_peak.test_get_num_recorded_errors(ErrorType::InvalidValue));
+    graphics::checkerboard_peak.test_get_num_recorded_errors(
+        ErrorType::InvalidValue,
+        None
```

Comment:
What's this second param for? counter and boolean don't have it in Rust (that I can see)

Reply:
haha, see. I made a note in my `notes.txt` to go back and fix that everywhere:
`test_get_num_recorded_errors` takes as the second parameter the (optional) ping to look into because ... well, technically you can look in other pings (and on Rust we can't leave it out).

Now the thing is: we don't actually need that: errors _always_ also go to the `metrics` ping on top of whatever ping this metric normally goes in. So we could be safe just removing that and always assuming `metrics`.
That's a bigger change though, so I'm filling a bug for that.
And because of that I'm less motivated to fix it everywhere just to revert it soon after.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1771705